<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>861</width>
     <height>179</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="7">
     <widget class="QSpinBox" name="sec_max">
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="week_label">
      <property name="text">
       <string>星期</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="t_long_label">
      <property name="text">
       <string>时长</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QSpinBox" name="sec_period">
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="period_label">
      <property name="text">
       <string>间隔</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="doy_label">
      <property name="text">
       <string>DoY</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QSpinBox" name="minu_min">
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="mon_label">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="mon_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSpinBox" name="hour_max">
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="day_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="year_min">
      <property name="minimum">
       <number>1970</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>2020</number>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QSpinBox" name="sec_min"/>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="week_min">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="day_min">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QSpinBox" name="minu_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="mon_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="mon_min">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="day_max">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
      <property name="value">
       <number>31</number>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QSpinBox" name="hour_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="year_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="minu_label">
      <property name="text">
       <string>分</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QSpinBox" name="minu_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QSpinBox" name="week_max">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="year_max">
      <property name="minimum">
       <number>1970</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2030</number>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QSpinBox" name="hour_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="hour_label">
      <property name="text">
       <string>时</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="min_label">
      <property name="text">
       <string>最小</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSpinBox" name="hour_min">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="max_label">
      <property name="text">
       <string>最大</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QSpinBox" name="sec_long">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QSpinBox" name="week_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QSpinBox" name="minu_max">
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="year_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="sec_label">
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="day_period">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="day_label">
      <property name="text">
       <string>日</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="mon_max">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="year_label">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="sod_label">
      <property name="text">
       <string>SoD</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QSpinBox" name="doy_period">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>366</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QSpinBox" name="doy_max">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>366</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QSpinBox" name="doy_min">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>366</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QSpinBox" name="sod_min">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>86400</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QSpinBox" name="sod_max">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>86400</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QSpinBox" name="sod_period">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>86400</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="recid_label">
      <property name="text">
       <string>rec_id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="typeid_label">
      <property name="text">
       <string>type_id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="recid_spin">
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="typeid_spin">
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="name_comb">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_to_db">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加/更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="list_out_tasks">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>列出任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_tasks">
   <property name="geometry">
    <rect>
     <x>699</x>
     <y>230</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>清除任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_db">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>270</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
